### DISCUSSION TITLE
Using Github Discussions moving forward

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#10] [General]
# Connected!
Today I'm please to announce that we now have a link between Github Discussions and Slack!

New Discussions and comments from this Github repository (json-schema-org/community) will be posted in the #ghd-community JSON Schema Slack server.

The GitHub Slack app doesn't yet have this functionality, and webhooks for GitHub Discussions is in beta, and so we had to build our own solution.

After being roadblocked deploying a code based solution, I ended up option for no-code solutions provider Zapier. The Zapier account is covered by Postman.

Currenly I'm the only one who has access to modify or update this, but given the temporary nature I don't consider this a huge issue. Once the GitHub Slack app supports Discussions, we can move to using that solution.

---

The display is similar to the #github channel, although using the basic setup with Zapier, you can only send messages as opposed to manually crafting the payload. Slack does have "blocks" for greater control over attachment display, but this would require more work as I would need to likely write some code in Zapier to construct the JSON. This isn't difficult, but setting this up has already taken longer than it should have done.

---

# What now?

Please feed back on the integration to me directly. We can start a discussion on future plans, which is what I'll be starting with next week.

If you have any ideas about how we can use Discussions, please also share them by creating a new Discussion!

**Comment by Relequestual** [comment]
Looks like there is some issues still to be worked out with the workflow... I'll get on it!

**Reply by Relequestual** [reply]
Issues with the connection should be resolved now! =]

**Comment by peteroneilljr** [comment]
@Relequestual howdy! I'm curious if you would be able to share the Zap you've created? The GitHub integration for Slack officially supports GHD, but there are some fine-grain controls that don't exist yet. It seems probable that I could do them with Zapier, but having a base to start from would save a ton of time. 

Thanks! 
Peter
